src/components/subscriptions/expiration/SubscriptionExpirationBanner.jsx

```diff
@@ -1,35 +1,69 @@
 import React, { useContext, useState } from 'react';
-import { MailtoLink, Alert } from '@edx/paragon';
+import { Button, Alert } from '@edx/paragon';
+import PropTypes from 'prop-types';
 
 import {
   SUBSCRIPTION_DAYS_REMAINING_MODERATE,
   SUBSCRIPTION_DAYS_REMAINING_SEVERE,
   SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL,
 } from '../data/constants';
 import { SubscriptionDetailContext } from '../SubscriptionDetailContextProvider';
+import { configuration } from '../../../config';
+import { formatTimestamp } from '../../../utils';
 
-const SubscriptionExpirationBanner = () => {
-  const { subscription: { daysUntilExpiration }, hasMultipleSubscriptions } = useContext(SubscriptionDetailContext);
+const SubscriptionExpirationBanner = (props) => {
+  const {
+    subscription: { daysUntilExpiration, expirationDate },
+    hasMultipleSubscriptions,
+  } = useContext(SubscriptionDetailContext);
   const [showBanner, setShowBanner] = useState(true);
 
-  const renderMessage = () => (
-    <>
-      {hasMultipleSubscriptions && daysUntilExpiration <= 0 ? (
+  const renderMessage = () => {
+    // use subscription detail view messaging
+    if (props.subscriptionDetails) {
+      return (
         <>
-          This subscription cohort has expired. You may still view the statuses of learners who participated.
+          {hasMultipleSubscriptions && daysUntilExpiration <= 0 ? (
+            <>
+              <Alert.Heading>
+                This subscription plan&apos;s end date has passed
+              </Alert.Heading>
+              Administrative actions are no longer available as of the plan end date of May 31, 2021.
+              You may still view the statuses of your invited learners.
+            </>
+          ) : (
+            <>
+              <Alert.Heading>
+                This subscription plan&apos;s end date is approaching
+              </Alert.Heading>
+              Administrative actions will no longer be available fallowing the
+              plan end date on {formatTimestamp({ timestamp: expirationDate })}.
+            </>
+          )}
```

Comment:
What is `hasMulitpleSubscriptions` is false but `daysUntilExpiration <= 0` is true?